src/components/pages/room.less

```diff
@@ -5,25 +5,22 @@
 .RoomPage-subtitle {
   display: flex;
   font-size: 12px;
-  gap: @edu-controls-group-gap;
   justify-content: space-between;
-  padding: 0 @edu-content-padding;
   color: @edu-text-color-secondary;
+  padding: 0 @edu-content-padding;
 }
 
-.RoomPage-subtitleGroup {
-  display: flex;
-  gap: 7px;
-  align-items: center;
+.RoomPage-leaveRoomLink {
+  .m-control-group;
+  padding-left: @edu-control-spacing-wide;
```

Comment:
The "leave room" link is no longer visible 🤔 

How it looks on Int:
<img width="511" alt="Screen Shot 2022-08-31 at 17 29 09" src="https://user-images.githubusercontent.com/8189923/187718329-40919deb-56eb-445c-bb60-d31f8c0e72d6.png">

How it looks locally:
<img width="522" alt="Screen Shot 2022-08-31 at 17 30 02" src="https://user-images.githubusercontent.com/8189923/187718397-de2cf63e-0b83-439b-a72c-915d9d5ea849.png">

Reply:
Ha! I reverted the condition in code in order to see the link while developing and forgot to set it back. Thanks for spotting it, it's fixed now.